src/Analysis/AnalysisApp.jsx

```diff
@@ -146,7 +160,11 @@ class AnalysisApp extends React.Component {
               className='analysis-app__iframe'
               title='Analysis App'
               frameBorder='0'
-              src={`${this.state.app.applicationUrl}`}
+              src={
+                this.state.app.title === 'OHDSI Atlas'
```

Comment:
```suggestion
                this.state.app.title === 'OHDSI Atlas' && this.state.app.needsTeamProject
```
I just realized we have this extra `needsTeamProject` option now.